addons/wp/stringtable.xml

```diff
@@ -3,777 +3,916 @@
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_MovingCenter_DisplayName">
             <English>Use group as center</English>
+            <Czech>Použít skupinu jako středobod</Czech>
             <German>Gruppe als Mittelpunkt benutzen</German>
             <Polish>Użyj pozycji grupy jako centrum</Polish>
             <Russian>Использовать группу как центр</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_PlayersOnly_DisplayName">
             <English>Group only targets players</English>
+            <Czech>Skupina cílí pouze na hráče</Czech>
             <German>Gruppe nimmt nur Spieler ins Visier</German>
             <Polish>Grupa będzie kierować się tylko do graczy.</Polish>
             <Russian>Группа нацелена только на игроков</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_PlayersOnly_Tooltip">
             <English>The hunting group will only target players. Disable to let the group target any hostile unit within range.</English>
+            <Czech>Lovecká skupina bude cílit pouze na hráče. Vypnutí nechá skupinu cílit na jakékoliv nepřátelské jednotky v dosahu.</Czech>
             <German>Die jagende Gruppe wird nur Spieler ins Visier nehmen. Durch Ausschalten wird die Gruppe alle feindlichen Einheiten in Reichweite ins Ziel nehmen.</German>
             <Polish>Polująca grupa będzie atakować tylko graczy. Wyłączenie opcji sprawi, że grupa będzie atakować każdą dowolną wrogą jednostkę w zasięgu.</Polish>
             <Russian>Группа охоты будет нацелена только на игроков. Отключите, чтобы позволить группе нацеливаться на любой враждебный юнит в пределах досягаемости</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskHunt_MovingCenter_Tooltip">
             <English>The hunting group will use the group leader as a center for the search pattern. Disable to have the unit use the module position instead. This creates a more defensive group</English>
+            <Czech>Nabíhající skupina použije velitele skupiny jako středobod pro pátrací vzor. Při vypnutí místo toho jednotka použije poziční modul. Toto vytvoří více obrannou skupinu</Czech>
             <German>Die jegende Gruppe benutzt den Gruppenführer als Mittelpunkt für das Suchmuster. Durch Ausschalten benutzt die Gruppe die Modulposition als Mittelpunkt. Das erzeugt eine defensivere Gruppe.</German>
             <Polish>Grupa użyje lidera jako centrum wzoru wyszukiwania. Wyłączenie opcji sprawi, że jednostka użyje zamiast tego pozycji modułu, co skutkuje tym, że grupa jest bardziej defensywna.</Polish>
             <Russian>Группа поиска будет использовать лидера группы в качестве центра схемы поиска. Отключите, чтобы устройство использовало вместо этого позицию модуля. Это создает более оборонительную группу</Russian>
-        </Key>
+        </Key>aaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
aaaaaaaaaaaaaaaaaaaaaaa?

Reply:
Severe issue of AAAAAAAA has been fixed.